### DISCUSSION TITLE
Which version Inter uses when we import it in our project via CDN??

### DISCUSSION THREAD

**Discussion by KunalTanwar11** [rsms/inter#524] [Q&A] [answered]
For instance when we download Inter we have 4 folders **namely** 

```
Inter Desktop
Inter Hinted for Windows
  - Desktop
  - Web
Inter Variable
Inter Web
```

So, I want to know when we import Inter via CDN is it uses `Inter Hinted for Windows\Web` or `Inter Web` ??

**Comment by rsms** [comment, marked as answer]
The CDN distro uses TTF files (in WOFF2 form) without hints. The release version on the CDN is always the [latest stable release as per GitHub](http://github.com/rsms/inter/releases/latest)
